pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -1139,6 +1143,23 @@ public void writeBrokerDataOnZooKeeper(boolean force) {
         }
     }
 
+    /**
+     * sort bundles by load and select topK bundles for each broker.
+     * @return the number of bundles selected
+     */
+    private int selectTopKBundle() {
+        bundleArr.clear();
+
+        bundleArr.addAll(loadData.getBundleData().entrySet());
+        // select topK bundle for each broker, so select topK * brokerCount bundle in total
+        int brokerCount = loadData.getBrokerData().size();
```

Comment:
can brokerCount be 0? are we handling this case?

Reply:
fixed.